src/questions/pancakeswap.yaml

```diff
@@ -0,0 +1,171 @@
+    - content: A stock exchange
+      key: C
+    - content: A smart contract exchange
+      key: D
+- uuid: edafdadd-c346-4347-9d92-895271d5e506
+  type: SingleChoice
+  content: What is Binance Smart Chain (BSC)?
+  hint: A blockchain network.
+  explanation: It is a blockchain network developed by Binance.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: A decentralized exchange
+      key: A
+    - content: A blockchain network developed by Binance
+      key: B
+    - content: A blockchain network developed that uses Ethereum
+      key: C
+    - content: A cryptocurrency trading platform
+      key: D
+- uuid: 598ed5b3-8ec1-41a7-af7b-f7f1ef34b6ca
+  type: MultipleChoice
+  content: What is the main benefit of Binance Smart Chain (BSC) compared to Ethereum?
+  hint: Less secure and lower user base.
+  explanation: BSC has faster transaction speeds and lower fees.
+  answerKeys:
+    - A
+    - B
+  difficultyLevel: Medium
+  choices:
+    - content: BSC has faster transaction speeds
+      key: A
+    - content: BSC has lower fees
+      key: B
+    - content: BSC is more secure
+      key: C
+    - content: BSC has a larger user base
+      key: D
+- uuid: 95aa3f9b-1ce8-453e-995d-46e9d75fa1e5
+  type: SingleChoice
+  content: Which of the following is NOT a feature of Pancakeswap?
+  hint: NoHint
+  explanation: Atomic swaps is not a feature of PancakeSwap.
+  answerKeys:
+    - D
+  difficultyLevel: Medium
+  choices:
+    - content: Yield farming
+      key: A
+    - content: NFT marketplace
+      key: B
+    - content: Initial farm offerings
+      key: C
+    - content: Atomic swaps
+      key: D
+- uuid: 82a073a7-cd33-4ecf-ae5d-95003a2f0b55
+  type: SingleChoice
+  content: Can users buy, sell, and trade non-fungible tokens (NFTs) on Pancakeswap?
+  hint: All are acceptable.
+  explanation: Users can buy, sell, and trade any NFT on Pancakeswap.
+  answerKeys:
+    - C
+  difficultyLevel: Low
+  choices:
+    - content: No, NFTs cannot be traded on Pancakeswap
+      key: A
+    - content: Yes, but only if they are created and managed through the Pancakeswap platform
+      key: B
+    - content: Yes, users can buy, sell, and trade any NFT on Pancakeswap
+      key: C
+    - content: Yes, but only if they are listed on the NFT Marketplace
+      key: D
+- uuid: 79c94035-0cc4-40e3-a5cb-839dd2a2dd39
+  type: SingleChoice
+  content: What consensus mechanism is used on Binance Smart Chain (BSC)?
+  hint: NoHint
+  explanation: Binance Smart Chain (BSC) uses a proof-of-staked-authority consensus mechanism. 
+  answerKeys:
+    - C
+  difficultyLevel: Medium
+  choices:
+    - content: Proof-of-work
+      key: A
+    - content: Proof-of-stake
+      key: B
+    - content: Proof-of-staked-authority
+      key: C
+    - content: Proof-of-elapsed-time
+      key: D
+- uuid: 1868960a-bc41-424c-b122-2e189c35c115
+  type: SingleChoice
+  content: What is the native cryptocurrency of Binance Smart Chain (BSC)?
```

Comment:
We should ask questions related to Pancakeswap instead of Binanance. Also we are a but pro-eth :-)